ansys/mapdl/core/launcher.py

```diff
@@ -863,7 +865,16 @@ def launch_mapdl(
 
     license_server_check : bool, optional
         Check if the license server is available if MAPDL fails to
-        start.  Only available on ``mode='grpc'``. Defaults to True
+        start.  Only available on ``mode='grpc'``. Defaults ``True``.
+
+    license_type : str, optional
+        Enable license type selection. You can input a string for its
+        license name (for example 'meba' or 'ansys') or its description
+        ("enterprise solver" or "enterprise" respectively).
+        You can also use legacy licenses (for example 'aa_t_a') but it will
+        also raise a warning. If it is not used (None), no specific license
+        will be requested, being up to the license server to provide a specific
+        license typ. Default is ``None``.
```

Comment:
```suggestion
        license name (for example ``'meba'`` or ``'ansys'``) or its description
        ("enterprise solver" or "enterprise" respectively).
        You can also use legacy licenses (for example ``'aa_t_a'``) but it will
        also raise a warning. If it is not used (``None``), no specific license
        will be requested, being up to the license server to provide a specific
        license type. Default is ``None``.
```